knowledge-base/datagrid-cell-doubletap.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Execute CellTap command on CellDoubleTap in DataGrid
+description: Execute the CellTap command in CellDoubleTap command in the RadDataGrid for .NET MAUI.
+type: how-to
+page_title: Execute the CellTap command in CellDoubleTap command in DataGrid
+slug: datagrid-cell-doubletap
+position: 
+tags: 
+ticketid: 1611664
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+        <tr>
+            <td>Product Version</td>
+            <td>5.2.0</td>
+        </tr>
+        <tr>
+            <td>Product</td>
+            <td>DataGrid for MAUI</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+This how-to article describes how to execute the `CellTap` command in `CellDoubleTap` command in the RadDataGrid for .NET MAUI.
```

Comment:
Can you elaborate more? This is the purely technical aspect of the issue. Always consider the developer's perspective:
* "I added a DataDrid and wanted to use the selection of an element with a double tap."
* "The TapGestureRecognizer is not working as expected, because it only gets recognized when you double tap on the Control, like the DataGrid header, but not on a single element."

After you add the developer's perspective, you will be able to revise the title and description of the KB, so you can include keywords that will be relevant to Google searches.